Description: Fix typos in man pages
 This patch fix some typos in two man pages
Author: Gennaro Oliva <oliva@debian.org>
Forwarded: https://support.schedmd.com/show_bug.cgi?id=21588
Last-Update: 2024-12-07

--- a/doc/man/man5/slurm.conf.5
+++ b/doc/man/man5/slurm.conf.5
@@ -5780,7 +5780,7 @@
 
 .TP
 \fBfm_url\fR
-If set, slurm will use the configured URL to interace with the fabric
+If set, slurm will use the configured URL to interface with the fabric
 manager to enable Slingshot hardware collectives.
 Note \fBenable_stepmgr\fR needs to be set for hardware collectives to run.
 .IP
--- a/doc/man/man8/slurmrestd.8
+++ b/doc/man/man8/slurmrestd.8
@@ -82,7 +82,7 @@
 
 .TP
 \fBlatest\fR
-Automatically replaced with latest plugin version. See relavent plugin for
+Automatically replaced with latest plugin version. See relevant plugin for
 description and potential flags.
 .IP
 
